terraform/files/sumocollector.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Send Habitat service logs to Sumologic syslog collector
+
+[Service]
+TimeoutStartSec=0
+ExecStart=/bin/sh -c '/bin/journalctl -fu hab-sup | /bin/ncat --udp localhost 1514'
```

Comment:
It might be worth making this unit dependent on the `hab-sup.service` unit--that way this won't get started earlier. Then again, `journalctl` will display nothing until the service shows up (if ever).

Reply:
Good point - unfortunately the dependency options are not _quite_ what we need. The closest one would be adding a 'Before=' flag, however that would have to be added to the hab-sup.service instead of the sumocollector.service.  If we run into startup issues, I'll look at doing that.